invenio_files_rest/config.py

```diff
@@ -127,3 +129,10 @@
 
 FILES_REST_FILE_TAGS_HEADER = 'X-Invenio-File-Tags'
 """Header for updating file tags."""
+
+FILES_REST_OFFLOAD_STREAMING_EXTERNALLY = False
```

Comment:
I would say that in general this feature is known as `X-Sendfile`, but it would be nice if you can check this.
`X-Accel-Redirect` is also something only for Nginx? I guess that we would like to provide something more generic, maybe with a default implementation for Nginx. What do you think?